README.md

```diff
@@ -137,7 +137,7 @@ $ pip install -r requirements.txt
 
 Run app.py from shell or command line. Note that the port needs to match what you've set up in your redirect_uri:
 ```Shell
-$ flask run --port 5000

```

Comment:
But the existing way matches what the [official Flask documentation](https://flask.palletsprojects.com/en/1.1.x/quickstart/?highlight=externally%20visible%20server#a-minimal-application) says. And it works for us when testing on `localhost`. We are testing it with the latest flask 1.1.1, by the way.

```
$ flask --version
Python 3.7.2
Flask 1.1.1
Werkzeug 0.15.2

$ flask run -p 8080
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
```

Could it be possible that you were experiencing some other issues not really affected by this `-h localhost`?

Reply:
I believe I was. Thanks for reviewing this change with me. I am now running my Flask app through a vscode launch json and it's working well for my development needs.

```js
{   
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python Console Daemon",
            "type": "python",
            "request": "launch",
            "program": "${workspaceFolder}/GuidesEncoder/app/__main__.py",
            "console": "integratedTerminal"
        },
        {
            "name": "Python: Flask Web App",
            "type": "python",
            "request": "launch",
            "module": "flask",
            "env": {
                "FLASK_APP": "${workspaceFolder}/GuidesEncoder/webapp/app.py",
                "FLASK_ENV": "development",
                "FLASK_DEBUG": "1"
            },
            "args": [
                "run"
            ],
            "jinja": true
        }
    ]
}
```